app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] == "cp":
+        raise ValueError("Bad command")
+    if command_parts[1] != command_parts[2]:
```

Comment:
The function should check if the command contains exactly three elements: 'cp', source file, and destination file. Add a condition to verify `len(command_parts) == 3`.